### ISSUE TITLE
Debugging with Visual Studio causes very slow mouse response

### ISSUE THREAD

**Issue by ianelletson** [rickbutton/workspacer#59]
This isn't really a bug per se, but when I try to debug this in visual studio, my mouse becomes very unresponsive until I hit a breakpoint and wait for a moment.

**Comment by rickbutton** [comment]
Yeah, I experience this sometimes too, which sucks. It is almost certainly because the message pump is suddenly left waiting for the workspacer process to response to an in-flight event, but it halts on a breakpoint, which lags mouse events for a while.

Not sure what the fix is for this, but it is annoying enough that I should really look into it.
